Examples/Scripts/Python/propagation.py

```diff
@@ -31,34 +40,62 @@ def runPropagation(trackingGeometry, field, outputDir, s=None, decorators=[]):
         propagatorImpl=prop,
         level=acts.logging.INFO,
         randomNumberSvc=rnd,
-        ntests=1000,
-        sterileLogger=True,
+        ntests=ntracks,
+        sterileLogger=sterile,
         propagationStepCollection="propagation-steps",
     )
 
     s.addAlgorithm(alg)
 
     # Output
-    s.addWriter(
-        acts.examples.ObjPropagationStepsWriter(
-            level=acts.logging.INFO,
-            collection="propagation-steps",
-            outputDir=outputDir + "/obj",
+    if not sterile:
+        if not os.path.exists(outputDir + "/obj"):
+            os.makedirs(outputDir + "/obj")
+        s.addWriter(
+            acts.examples.ObjPropagationStepsWriter(
+                level=acts.logging.INFO,
+                collection="propagation-steps",
+                outputDir=outputDir + "/obj",
+            )
```

Comment:
```suggestion
        outputDirObj = os.path.join(outputDir, "obj")
        if not os.path.exists(outputDirObj):
            os.makedirs(outputDirObj)
        s.addWriter(
            acts.examples.ObjPropagationStepsWriter(
                level=acts.logging.INFO,
                collection="propagation-steps",
                outputDir=outputDirObj,
            )
```
for better portability